roles/infrastructure/tasks/network.yml

```diff
@@ -21,23 +27,31 @@
     delegations: "{{ item.delegations | default(omit) }}"
   loop: "{{ infrastructure_vnet_subnets }}"
   register: aap_networking
+  tags:
+    - network
+    - database
 
 - name: Create public IP addresses for systems
```

Comment:
This was actually removed from my PR: https://github.com/ansible-content-lab/azure_ansible_deployment/pull/7/files#diff-44f33e6bf29bdf4ca38d0589b7ba5a7f0f5637f419a4ed4b85b3b2cc9d70fc06L25

We're creating the Public IP directly on the VM Create [playbook](https://github.com/ansible-content-lab/azure_ansible_deployment/pull/7/files#diff-d3e2490635830a4a3f42104984f3ec952d230089e6422d626689378f493118aeR26) , you may need to create a new task in the `database.yml` for creating the PUB IP.

Reply:
That was merely an improvement over the existing code. We don't really need the database to create any public IP address.